src/linux/cgroups.cpp

```diff
@@ -691,21 +760,6 @@ Try<Nothing> create(
 }
 
 
-Try<Nothing> remove(const string& hierarchy, const string& cgroup)
```

Comment:
Let's remove the declaration from the header file too.

Overall, LGTM. I'll run a CI build for this patch and merge it later.

Reply:
> Let's remove the declaration from the header file too

Done.